torch/csrc/jit/passes/quantization.cpp

```diff
@@ -661,8 +667,15 @@ void InsertQuantDeQuantHelper::collectObserverNodesAndValueToQuantize(
   Value* new_value = observer->input(1);
   v->replaceAllUsesWith(new_value);
   auto tp = getQSchemeAndQParamMap(module, v);
+  auto qscheme = std::get<0>(tp);
+  if (qscheme_for_graph_.count(g)) {
+    TORCH_CHECK(qscheme_for_graph_.at(g) == qscheme,
+                "Quantizing same graph with different QSchemes is not supported");
+  } else {
+    qscheme_for_graph_[g] = qscheme;
+  }
   auto qparam_map = std::get<1>(tp);
-  values_to_qparams_[g].insert({new_value, qparam_map});
+  values_to_qparams_[g][module._ivalue()].insert({new_value, qparam_map});
```

Comment:
This indexing by a module object seems weird. Can we just set the proper attribute in `module._ivalue()` and then use it? The map would then be to the name of the attribute I think.

Reply:
I realized that we shouldn't store qparam here, we should call `getQSchemeAndQParamMap` for each module instance instead